pdf-generator/src/test/java/uk/gov/hmcts/reform/cmc/pdf/generator/PDFGeneratorTest.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.hmcts.reform.cmc.pdf.generator;
+
+import org.junit.Test;
+import org.xml.sax.SAXParseException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchThrowable;
+
+public class PDFGeneratorTest {
+
+    private PDFGenerator pdfGenerator = new PDFGenerator();
+    private XMLContentSanitizer contentSanitizer = new XMLContentSanitizer();
+
+    @Test
+    public void shouldThrowSaxParseExceptionWhenGivenHTMLWithIllegalCharacters() {
```

Comment:
nice, this ensures that the test doesn't keep passing if someone accidentally removes the broken characters 👍